opentargets_tep.json

```
@@ -0,0 +1,46 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "OpenTargets-tep",
+  "description": "OpenTargets TEP (Target Enabling Package) objects",
+  "type": "object",
+  "properties": {
+    "targetFromSource" : {
+      "type": "string",
+      "description": "Target symbol/name as downloaded from the source",
```

```suggestion
      "description": "Target ID in resource of origin (accepted sources include Ensembl gene ID, Uniprot ID, gene symbol)",
```

To make it consistent with the fields in the evidence schema.